extensions/Lily/Assets.js

```diff
@@ -0,0 +1,630 @@
+// Name: Asset Manager
+// ID: lmsAssets
+// Description: Add, remove, and get data from sprites, costumes and sounds. 

```

Comment:
Minor nitpick, but the oxford commas in the description are inconsistent.

Reply:
damn you're right